contracts/btc-finality/tests/integration.rs

```diff
@@ -21,6 +21,7 @@ fn wasm_size_limit_check() {
 }
 
 #[test]
+#[ignore = "Unsupported query type: GRPC"]
```

Comment:
could you give more context about this one?

Reply:
The mocked environment in which integration tests are run don't support these queries. Since this query is now part of the finality contract instantiation, the instantiation integration test fails.

There's no easy way around this, except disabling or removing the test, as these integration tests are run against the optimised wasm binary that is going to be released.